### PR TITLE
telink: b92: update hal commit.

### PR DESCRIPTION
**Pull request by haiwentelink**
- update hal commit for hal .

### CODE REVIEW

**Comment by andriy-bilynskyy** [comment]
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/86febf1fe192375e16b7463f86ed69fa63ac3a6e | https://github.com/telink-semi/hal_telink/commit/fa259a5b472606e0d0e54be356929446257285b2 (telink_tl321x) | [telink-semi/hal_telink@86febf1f..fa259a5b](https://github.com/telink-semi/hal_telink/compare/86febf1fe192375e16b7463f86ed69fa63ac3a6e..fa259a5b472606e0d0e54be356929446257285b2) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

File: west.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ manifest:
         - hal
     - name: hal_telink
       url: https://github.com/telink-semi/hal_telink
-      revision: 86febf1fe192375e16b7463f86ed69fa63ac3a6e
+      revision: pull/101/head
```
Comment (andriy-bilynskyy) [inline west.yml]: Please update revision (hal already merged)

**Review by andriy-bilynskyy — CHANGES_REQUESTED** [review]

**Comment by haiwentelink** [comment]
@andriy-bilynskyy  CI have already passed , and commit id have already update .

**Review by andy-liu-telink — APPROVED** [review]
Approve

**Review by andriy-bilynskyy — APPROVED** [review]

**Pull request merged**